### PR TITLE
6.7.0: Release notes

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Release notes for the 6.7.0 release.

## Motivation and Context
Support 6.7.0 release

Signed-off-by: Hillary Fraley <hfraley@sumologic.com>

### CODE REVIEW

File: content/sensu-go/6.7/release-notes.md [hunk 1]
```diff
@@ -102,6 +103,42 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+- The [sensu.CheckDependencies][284] Sensu query expression now supports arrays of strings and arrays of objects.
+- On backend startup, Sensu now creates the [`sensu-system` namespace][279] and a [backend entity][278] to log secrets provider errors and help prevent spamming the event bus with backend events.
+- Connection and TLS error log entries now include a `source` property that lists the corresponding agent's IP address and port to identify which agent generated each log entry for troubleshooting.
+- Increased the default values for the backend configuration flags [etcd-election-timeout][272] (from 1000 to 3000) and [etcd-heartbeat-interval][273] (from 100 to 300).
+- Upgraded etcd version from `3.5.0` to `3.5.2`.
+
+**FIXES:**
+
+- ([Commercial feature][268]) Fixed a bug that could cause pipeline resources to hang when using a [TCP stream handler][283] whose `max_connections` attribute is set to greater than zero.
+- Fixed a bug that prevented accurate ARM version detection for sensu-agent.
```
Comment (hillaryfraley) [inline content/sensu-go/6.7/release-notes.md:142]:
```suggestion
- In events, event.check objects now include any pipelines specified in the check configuration.
- Socket handlers that are interrupted by an error mid-write will no longer cause a sensu-backend panic.
Also, socket handlers will now respect their timeout settings after the initial connection is established.
- Fixed a bug that prevented accurate ARM version detection for sensu-agent.
```

File: content/sensu-go/6.7/release-notes.md [hunk 2]
```diff
@@ -102,6 +103,45 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+- ([Commercial feature][268]) [Business service monitoring (BSM)][281] now uses the PostgreSQL round robin Ring V2 implementation, even if the [`enable_round_robin`][280] attribute is set to `false` in the PostgresConfig definition.
+- ([Commercial feature][268]) Added the `sensu_go_etcd_cluster_leases` metric to the backend [metrics log][282] to track the count of current etcd leases for debugging.
+- ([Commercial feature][268]) Added logging for [TCP stream handler][283] events.
+- The [sensu.CheckDependencies][284] Sensu query expression now supports arrays of strings and arrays of objects.
+- On backend startup, Sensu now creates the [`sensu-system` namespace][279] and a [backend entity][278] to log secrets provider errors and help prevent spamming the event bus with backend events.
+- Connection and TLS error log entries now include a `source` property that lists the corresponding agent's IP address and port to identify which agent generated each log entry for troubleshooting.
```
Comment (hillaryfraley) [inline content/sensu-go/6.7/release-notes.md]:
```suggestion
- For connections with faulty TLS configurations, error log entries now include a `source` property that lists the corresponding agent's IP address and port to identify which agent generated each log entry for troubleshooting.
```
```suggestion
- Connection and TLS error log entries now include a `source` property that lists the corresponding agent's IP address and port to identify which agent generated each log entry for troubleshooting.
```

Comment (hillaryfraley) [inline content/sensu-go/6.7/release-notes.md]:
```suggestion
- For connections with faulty TLS configurations, error log entries now include a `source` property that lists the corresponding agent's IP address and port to identify which agent generated each log entry for troubleshooting.
```
```suggestion
- Connection and TLS error log entries now include a `source` property that lists the corresponding agent's IP address and port to identify which agent generated each log entry for troubleshooting.
```

**Review by asachs01 — APPROVED** [review]
LTGM

**Pull request merged**